Library/Homebrew/utils/shell.rb

```diff
@@ -39,9 +39,15 @@ def export_value(key, value, shell = preferred)
 
     # return the shell profile file based on user's preferred shell
     def profile
-      return "#{ENV["ZDOTDIR"]}/.zshrc" if preferred == :zsh && ENV["ZDOTDIR"].present?
+      case preferred
+      when :bash
+        bash_profile = "#{ENV["HOME"]}/.bash_profile"
```

Comment:
Why is this case `.bash_profile` and not `.profile`?

Reply:
Use `~/.bash_profile` if the current shell is `/bin/bash` and the file `~/.bash_profile` already exists. When `bash` starts up, it looks for `~/.bash_profile` and uses it if it exists, and otherwise uses `~/.profile` if it exists.

Reply:
Ah gotcha, reread and makes sense 👍